src/main/java/com/sap/oss/phosphor/fosstars/advice/AdviceContentYamlStorage.java

```diff
@@ -71,7 +71,7 @@ protected AdviceContentYamlStorage(Map<Feature<?>, List<RawAdviceContent>> featu
    * @param context The context.
    * @return A list of advice.
    */
-  public List<AdviceContent> adviceFor(Feature<?> feature, AdviceContext context) {
+  public List<AdviceContent> advicesFor(Feature<?> feature, AdviceContext context) {
```

Comment:
I used to think that "advices" is a plural form of "advice". So I called this method `advicesFor`. But then I learned that "advice" is uncountable, therefore it has no plural form.

https://english.stackexchange.com/questions/282047/so-some-advice-or-some-advices-which-is-correct

Let's please revert this.
```suggestion
  public List<AdviceContent> adviceFor(Feature<?> feature, AdviceContext context) {
```